lib/helpers.js

```diff
@@ -243,6 +244,21 @@ exports.verifyDelegation = async ({capability, edvId}) => {
   }
 };
 
+exports.verifyDigestHeaderValue = async (req, res, next) => {
+  const expectedDigest = req.headers.digest;
+  const {verified} = await verifyHeaderValue({
```

Comment:
Looking at this, I wondered what would happen if `req.headers.digest` was undefined or otherwise not the expected data type.  So I looked up the API:

https://github.com/digitalbazaar/http-digest-header/blob/581fded1185dfb1b74066edcf466bea258e20426/lib/httpDigest.js#L39

Which shows me that `.split` is going to be potentially called on an undefined value here:
https://github.com/digitalbazaar/http-digest-header/blob/581fded1185dfb1b74066edcf466bea258e20426/lib/httpDigest.js#L66

I think the `verifyHeaderValue` needs some type validation.

cc: @dmitrizagidulin

Reply:
@mattcollier, @dmitrizagidulin -- currently, `verifyHeaderValue` will throw an error from within a try/catch which will result in `verified: false` being returned. We'll want to decide if that's sufficient here or not. We probably want a better `error` that says the digest header value is missing.

Reply:
+1 to a better error.  IMO, throwing on `cannot access split of undefined` is not a good user/dev experience.

Reply:
Created an issue: https://github.com/digitalbazaar/http-digest-header/issues/8